Sources/xcproj/PBXProj+Helpers.swift

```diff
@@ -41,15 +41,15 @@ extension PBXProj {
     /// - Parameter reference: reference of the file whose type will be returned.
     /// - Returns: String with the type of file.
     func buildPhaseType(buildFileReference: String) -> BuildPhase? {
-        if sourcesBuildPhases.contains(where: {$0.files.contains(buildFileReference)}) {
+        if objects.sourcesBuildPhases.contains(where: { k,v in v.files.contains(buildFileReference)}) {
```

Comment:
Nitpick: there's probably going to be a warning about not using `k`. It could either be `_, v` or what about just
```
if objects.sourcesBuildPhases.values.contains(where: { $0.files.contains(buildFileReference)}) {
```

Reply:
Same for all below this in different functions